creating_packages/package_dev_flow.rst

```diff
@@ -91,7 +92,7 @@ dependencies. It uses a path to a folder where it will perform the build.
 
 .. code-block:: bash
 
-    $ conan build . --source-folder=tmp/source --install-folder=tmp/install --build-folder=tmp/build
+    $ conan build . --source-folder=tmp/source --install-folder=tmp/install_build --build-folder=tmp/install_build
```

Comment:
This is redundant and feels a bit ugly. I think ``-install-folder`` defaults to the build folder, so remove it.

I will also change ``install_build`` => ``build``, and describe just the most natural way.